myriadeploy/deployment.cfg.local

```diff
@@ -16,3 +14,8 @@ rest_port = 8753
 [workers]
 1 = localhost:9001
 2 = localhost:9002
+
+[runtime]
+# Uncomment to set the maximum heap size of the worker processes
+#max_heap_size = -Xmx2g
```

Comment:
I almost suggested this too -- but if you do go this route, it might make sense to expose an additional "JVM flags" key since I know that people use this to push other command-line options.